### PR TITLE
Milestone1: Setup Dev and Production Environment

### PR DESCRIPTION
**Pull request by anewman15**
In this PR,

I started working on the Calculator app using React. I set up the dev and deployment env by completing the following steps:
- Initialized the React app
- Create an app on Heroku
- Deploy the main branch to Heroku using mars
- Follow Git Flow and continue development in milestone1-setup branch

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @anewman15 👋

**You've done quite well on this. Some unnecessary files have been removed. 👌**

However, let's make some fixes.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off",
+      "indent": [2, "tab"],
+      "no-tabs": 0,
+      "import/no-cycle": "off",
+      "no-restricted-globals": "off",
+      "no-alert": "off"
```
Comment (mikenath223) [inline .eslintrc.json]: - [x] Please use the same lint config rules in this [link](https://github.com/microverseinc/linters-config/tree/master/javascript). It's important you use the same eslint rules used by fellow students too.

**Comment by mikenath223** [comment]
- [x] Kindly remove logo.svg in src directory. It's unnecessary.

**Comment by mikenath223** [comment]
- [x] Please deploy the project and include the live link in the readme.

**Comment by anewman15** [comment]
@mikenath223 Thanks. Changes made

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this milestone approved. :+1:

You can merge this branch into the development branch when you're ready and go on to your next milestone.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Pull request merged**